### PR TITLE
style improvements in the instructor dashboard

### PR DESCRIPTION
**Pull request by muhhshoaib**
@chrisndodge review the changes for PHX-112 
style improvements for the Student attempts section in proctoring tab.

### CODE REVIEW

File: edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js [hunk 1]
```diff
@@ -5,6 +5,20 @@ var edx = edx || {};
...
+        started: 'Started',
+        ready_to_submit: 'Ready to submit',
+        declined: 'Declined',
+        timed_out: 'Timed out',
+        submitted: 'Submitted',
+        verified: 'Verified',
+        rejected: 'Rejected',
+        not_reviewed: 'Not reviewed',
+        error: 'Error'
+    };
```
Comment (chrisndodge) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js]: I'm thinking we should wrap all of these inside of gettext()'s to make internationalizable.

Comment (muhhshoaib) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js]: done

File: edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js [hunk 2]
```diff
@@ -5,6 +5,20 @@ var edx = edx || {};
...
+        ready_to_start: gettext('Ready to start'),
+        started: gettext('Started'),
+        ready_to_submit: gettext('Ready to submit'),
+        declined: gettext('Declined'),
+        timed_out: gettext('Timed out'),
+        submitted: gettext('Submitted'),
+        verified: gettext('Verified'),
+        rejected: gettext('Rejected'),
+        not_reviewed: gettext('Not reviewed'),
+        error: 'Error'
```
Comment (chrisndodge) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js]: missed one?

Comment (muhhshoaib) [inline edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js]: done

**Comment by chrisndodge** [comment]
+1 from me. I think you can merge this in by itself, but there's an open styling change in edx-platform. I'll tag Brian on that one as well.

**Comment by muhhshoaib** [comment]
edx-platform branch has been merged. It's good to go now.

**Pull request merged**